### PR TITLE
Updating and standardizing the license headers

### PR DESCRIPTION
**Pull request by keveleigh**
Overview
---
Updates, adds, and standardizes the license headers across the Toolkit.

### CODE REVIEW

File: Assets/HoloToolkit/License.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2016 Microsoft Corporation
+Copyright (c) 2017 Microsoft Corporation
```
Comment (StephenHodgson) [inline Assets/HoloToolkit/License.txt]: Should we also add the toolkit version here?

Comment (keveleigh) [reply Assets/HoloToolkit/License.txt]: Hmm yeah, either here or a separate version.txt. It probably makes sense to have it all-in-one.

Comment (NeerajW) [reply Assets/HoloToolkit/License.txt]: MRTK_Version.txt or something akin.

Comment (keveleigh) [reply Assets/HoloToolkit/License.txt]: I added MRTKVersion.txt to the HoloToolkit folder.

**Comment by keveleigh** [comment]
@StephenHodgson Conflicts resolved.

**Comment by StephenHodgson** [comment]
@keveleigh should we also do this for the stabilization branch?

You might be able to cherry pick that in.

**Comment by keveleigh** [comment]
I'd be happy to see it go into the stabilization branch, since it's a "doc" update rather than a feature update.

**Pull request merged**